s4e-web/src/assets/images/ico_drag.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Have you maybe think about using [Font Awesome Icons](https://fontawesome.com/icons) and tag <i></i> in your code? It's much easier to put into an application than loading SVG files into assets and loading it trough image src parameter.

Reply:
Yes i think about FA but:
1. I try to avoid inserting additional tag into html where is not necessary.   
2. Not always using <i> is easier than adding svg to css.

Besides I don't want to be forced to use it in my design as an only solution. ;)
Part of icons are made by myself where i don't have same "look and feel" design in FA.

Of course we can mixed solutions.

Reply:
ok